sql/pr/sith-lords.sql

```diff
@@ -0,0 +1,45 @@
+-- Always two, there are. No more. No less. A master and an apprentice
+WITH mergers_per_year AS (
+    SELECT
+        year(c.commit_time AT TIME ZONE 'UTC') AS year,
+        ai.name as ai_name,
+        regexp_replace(ai.email, '(?<=.)[^@](?=[^@]*?@)|(?:(?<=@.)|(?!^)\G(?=[^@]*$)).(?=.*\.)', '*') AS ai_email,
+        ma.org as ai_org,
+        ci.name as ci_name,
+        regexp_replace(ci.email, '(?<=.)[^@](?=[^@]*?@)|(?:(?<=@.)|(?!^)\G(?=[^@]*$)).(?=.*\.)', '*') AS ci_email,
+        mc.org as ci_org,
+        count(*) AS commits_count
```

Comment:
This is skewed by PRs having multiple commits, can you get the first commit grouped by time?